internal/attestation/crafter/testdata/contracts/with_policy_embedded.yaml

```diff
@@ -0,0 +1,4 @@
+schemaVersion: "v1"
+policies:
+  attestation:
+    - ref: testdata/contracts/policy_embedded.yaml
```

Comment:
is this embedded?

Reply:
The code in the policy, not the policy in the contract.